lib/ello_click/affiliate/link.ex

```diff
@@ -0,0 +1,9 @@
+defmodule ElloClick.Affiliate.Link do
+  defstruct [original: nil, affiliated: nil, is_affiliated: false]
```

Comment:
We were actually thinking this could even be taken a little further and have those be %URI{} structs so we could match on hostname without schema easier.